Library/Homebrew/keg_relocate.rb

```diff
@@ -97,7 +99,12 @@ def detect_cxx_stdlibs(_options = {})
   end
 
   def each_unique_file_matching(string)
-    Utils.popen_read("/usr/bin/fgrep", "-lr", string, to_s) do |io|
+    bsd = `/usr/bin/fgrep -V`.include?("BSD grep")
```

Comment:
> Why not backticks, out of curiosity? Consistency or something more subtle?

I dunno, it's just a pattern we've been following here for a while. I think we handle them slightly differently. I don't care enormously, to be honest.

> I think I'd have to copy the body of the method into both os/mac/keg_relocate.rb and os/linux/keg_relocate.rb and just change the arguments that are being passed to grep. Is that right? It feels repetitive.

Ideally use a separate `grep_arguments` method for just getting the arguments which should make it pretty small. Up to you to decide what the `generic_grep_arguments` version should be but I think there's only a need for a single override e.g. assume BSD here and override in `os/linux` or assume GNU here and override in `os/mac`.